common/app/views/fragments/analytics/google.scala.html

```diff
@@ -30,8 +30,8 @@
     }
 
     function getAnalyticsTitle() {
-        var analyticsTitle = (guardian.config.page.webTitle || '').toLowerCase();
-        return (analyticsTitle === 'network front') ? getAnalyticsEdition() + ' ' + analyticsTitle : analyticsTitle;
+        var analyticsTitle = (guardian.config.page.webTitle || '');
+        return (analyticsTitle === 'Network Front') ? getAnalyticsEdition() + ' network front' : analyticsTitle;
```

Comment:
Is 'network front' lowercased here consistent with apps then or should that be left as 'Network Front'?